USAGE.md

```diff
@@ -267,14 +267,45 @@ Policies Leaf NS - - - - - - - - >           group
 (managed by NS user)
 ```
 
-## How to Grant User based Access for LDAP/OIDC?
+## How to Grant User based Access for LDAP/OIDC/JWT?
 
 Example Terraform code is given in [`./docker/terraform/vault-identity-ldap-oidc-user-access.tf`](./docker/terraform/vault-identity-ldap-oidc-user-access.tf).
 
 The LDAP user `kv-writer` is granted read/write privilges for the KV engine in the `tenant` namespace.
 
 The access is only granted for LDAP logins, because the Entity Alias is connected to the LDAP auth backend only.
 
+## How to login using JWT token authentication from CLI?
+This playground also has a JWT authentication backend available. Configuration is basically the same as for the OIDC authentication backend. The difference is that a token fetched from keycloak can be used directly to log into Vault.
+
+Fetch token
+```bash
+username=test
+password=test
+curl -s -L -X POST "http://keycloak.${CONTAINER_DOMAIN}:8080/auth/realms/${CONTAINER_DOMAIN//\./-}/protocol/openid-connect/token" \
+-H 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'client_id=hashicorp-vault-jwt' \
+--data-urlencode 'grant_type=password' \
+--data-urlencode 'scope=openid' \
+--data-urlencode "username=${username}" \
+--data-urlencode "password=${password}" > jwt.json
```

Comment:
fixed the variable here to use `password` in 59686e4d38c1bcbbfd3f59eede07559417105867